docs/flow/modin/engines/ray/cudf_on_ray/frame/partition.rst

```diff
@@ -0,0 +1,17 @@
+cuDFOnRayFramePartition
+"""""""""""""""""""""""""
+
+The class is specific implementation of ``BaseFramePartition``, providing an API
+to perform an operation on a block partition, namely, cudf.DataFrame, using Ray as an execution engine.
+
+An operation on a block partition can be performed in two modes:
+
+* asyncronously - via :meth:`~modin.engines.ray.cudf_on_ray.frame.cuDFOnRayFramePartition.apply`
+* lazily - via :meth:`~modin.engines.ray.cudf_on_ray.frame.cuDFOnRayFramePartition.add_to_apply_calls`
```

Comment:
This is not true. Both `apply` and `add_to_apply_calls` are performed asynchronously, but their return values are different.

Reply:
Section was removed. The same sentences are used in `docs/flow/modin/engines/ray/pandas_on_ray/frame/partition.rst`.

Reply:
I do not think this needs to be removed at all. Probably, it would be good to rephrase it a bit.

> The same sentences are used in `docs/flow/modin/engines/ray/pandas_on_ray/frame/partition.rst`.

Behaviors of `apply` and `add_to_apply_calls` for `PandasOnRay` and `cuDFOnRay` are different.

Reply:
rephrased